socranop/data/man/socranop-ctl.1

```diff
@@ -0,0 +1,160 @@
+.\" ======================================================================
+.\"
+.\" The socranop-ctl(1) man page
+.\"
+.\" This man page has been (re)written adhering to the following
+.\" documentation: man(7), man-pages(7), tbl(1)
+.\"
+.\" ======================================================================
+.\"
+.TH SOCRANOP\-CTL 1 "2021\-07\-04" "${PACKAGE} ${VERSION}" "User commands"
+.\"
+.\" ======================================================================
+.\"
+.SH NAME
+socranop\-ctl \- control Soundcraft Notepad series mixer USB audio routing from command line
+.\"
+.\" ======================================================================
+.\"
+.\" usage: socranop-ctl [-h] [--version] [--no-dbus] [-l] [-s SET]
+.\"
+.\" optional arguments:
+.\"   -h, --help         show this help message and exit
+.\"   --version          show program's version number and exit
+.\"   -v, --verbose      Enable more verbose output, largely for debugging
+.\"   --no-dbus          Use direct USB device access instead of D-Bus service
+.\"                      access
+.\"   -l, --list         List the available source routing options
+.\"   -s SET, --set SET  Set the specified source to route to the USB capture
+.\"                      input
+.\"
+.SH SYNOPSIS
+.B "socranop\-ctl"
+.RB (\| \-\-help \|| \-\-version \|)
+.br
+.B "socranop\-ctl"
+.RB [\| \-\-verbose \|]
+.RB [\| \-\-no\-dbus \|]
+.B \-\-list
```

Comment:
Should we give the short option name too?

Reply:
I don't think the SYNOPSIS absolutely needs the short options if there are long options available which explain what the option does.

The short options are documented as equivalent in the OPTIONS section below. If someone cannot make that connection, I am not very sorry for them typing a few letters extra. Anyway, the bash completion will also help them type if they happen to run bash.